src/@utils/aquarius.ts

```diff
@@ -164,6 +165,31 @@ export async function retrieveAsset(
     }
   }
 }
+let schema: any
+export function retrieveShaclSchema(): Promise<ShaclSchema> {
+  console.log(schema)
```

Comment:
You can remove this log

Reply:
yeah, some how I didn't push my last commit with these cleanup changes. But thanks for pointing this out.